modules/stochastic_tools/src/utils/GaussianProcess.C

```diff
@@ -76,7 +80,8 @@ GaussianProcess::setupCovarianceMatrix(const RealEigenMatrix & training_params,
   _K.resize(_num_outputs * _batch_size, _num_outputs * _batch_size);
 
   if (_tuning_data.size())
-    tuneHyperParamsAdam(training_params, training_data, opts);
+    // tuneHyperParamsAdam(training_params, training_data, opts);
```

Comment:
You must provide an option to either use Adam or MCMC. The default must me Adam. You need to add this option in the `GaussianProcessTrainer` class as a new tuning option and pass the updated tuning options to this class.